config/schema/synonyms.yml

```diff
@@ -53,46 +61,46 @@
 - search: emissions, pollution
 - search: fees, costs, prices
 - search: reduction, discount
-- search: renew, expired, out of date
-- search: self employed, self employment, sole trader, starting a business
+- search: expired, out of date => expired, out of date, renew
+- search: self employed, self employment, sole trader
 
 # Adding terms to return relevant content that doesn't mention the original search term
 - search: how much => how much, fees, rates
 - search: apostille => apostille, document legalised
-- search: bad weather, winter weather => cold weather, severe weather, winter weather, energy grants, heating
+- search: bad weather, winter weather => bad weather, cold weather, severe weather, winter weather, energy grants, heating
 - search: benefit checker, benefits checker => benefits adviser
 - search: bin collection => bin collection, rubbish collection
-- search: bno => overseas british passport
+- search: bno => bno, overseas british passport
 - search: cancel child benefit => stop child benefit
 - search: curriculum vitae => cv, curriculum vitae
 # Seasonal need - also for Christmas and New Year payments
 - search: easter => easter, paid early
-- search: fiance, fiancee, fiancé, fiancée => partner, family, visa
+- search: fiance, fiancee, fiancé, fiancée => fiance, fiancee, fiancé, fiancée, partner, family, visa
 - search: flood alert, flood alerts => flood alerts, flood warnings
 - search: hpi check => used car check
 - search: intestacy => intestacy, wills, inherits
 - search: intestate => intestate, wills, inherits
 - search: kiev => kiev, kyiv, ukraine
 - search: mot checker => mot check
 # Student finance login
-- search: my account => your account, online account
+- search: my account => my account, your account, online account
 - search: not arrived => not arrived, contact dvla
 - search: not received => not received, contact dvla
 - search: refuse collection => refuse collection, rubbish collection
-- search: self certificate => self certification
+- search: self certificate => self certificate, self certification
```

Comment:
would this one be covered by normal stemming rules?
Also, would this add self to the resulting analysed sentence twice, artificially increasing its relevance?

Reply:
For whatever reason, `certificate` stems to `certif` and `certification` stems to `certificat`